### DISCUSSION TITLE
weppy with python38

### DISCUSSION THREAD

**Discussion by josejachuf** [emmett-framework/emmett#367] [General]
I'm trying to run weppy in an venv with python38, I fixed an import and weppy start, but I get in browser:
**Server Error** 

Apparently the problem is in rocket.py. Has someone been able to run weppy with python38? 

Jose

**Comment by gi0baro** [comment]
@josejachuf I'm quite sure you will have issues, but my suggestion is to deactivate reloader when you run the app

**Reply by josejachuf** [reply]
There are many things that have changed and have been deprecado in PY38. I was already fixing several, the app starts, but there are things that tasted faults. If I can make it walk, send the pull request
